src/qilib/utils/storage/mongo.py

```diff
@@ -208,6 +262,43 @@ def save_data(self, data: Any, tag: TagType) -> None:
         StorageInterface._validate_tag(tag)
         self._store_value_by_tag(tag, self._encode_data(self._serialize(data)), self._get_root())
 
+    def load_individual_data(self, field: Any, tag: TagType) -> Any:
+        """ Retrieve an individual field value at a given tag
+
+            Args:
+                field: Name of the individual field to be retrieved
+                tag: The tag
+
+            Returns:
+                Value of the field
+        """
+
+        StorageInterface._validate_tag(tag)
```

Comment:
This is not the normal way to call a protected member of a base class. Unless we do it by purpose (but I don't see it) we should use self._validate_tag.
Please do some boy scouting here because I see more instances of this construct (also in memory.py)